sdcm/argus_results.py

```diff
@@ -128,6 +128,7 @@ class Meta:
             ColumnMetadata(name="l&s bandwidth", unit="MiB/s/shard", type=ResultType.FLOAT, higher_is_better=True),
             ColumnMetadata(name="repair time", unit="s", type=ResultType.DURATION, higher_is_better=False),
             ColumnMetadata(name="total", unit="s", type=ResultType.DURATION, higher_is_better=False),
+            ColumnMetadata(name="size", unit="GB", type=ResultType.FLOAT, higher_is_better=False),
```

Comment:
if this size is not a subject of change, ValidationRule should be added.
Probably also `higher_is_better` should be set to true.
This way, when this size changes, test will fail.